pom.xml

```diff
@@ -63,6 +64,18 @@
                 <artifactId>spring-boot</artifactId>
                 <version>${spring.version}</version>
             </dependency>
+            <dependency>
```

Comment:
These shouldn't be here
They're specific to the email-service.

If you do want to add them here then their version should be declared here (parent pom) and the email-service pom should not have any versions for them. That's how dependencyManagement works